extension/tasks/dependabotV2/utils/getSharedVariables.ts

```diff
@@ -160,6 +165,9 @@ export default function getSharedVariables(): ISharedVariables {
   let commentPullRequests: boolean = tl.getBoolInput('commentPullRequests', false);
   let abandonUnwantedPullRequests: boolean = tl.getBoolInput('abandonUnwantedPullRequests', true);
 
+  let dependabotCliImage: string | undefined = tl.getInput('dependabotCliImage');
+  let dependabotUpdaterImage: string | undefined = tl.getInput('dependabotUpdaterImage');
```

Comment:
Do we need to ensure that the updater image contains "{ecosystem}" in it?
If we do not, we can assume the user will know they messed up something.

Otherwise, I like it being a single input as compare to one for tag and another for the registry like I had in v1. This single input can work for those who always want to have their own container registry.

Reply:
Yes, good point. I will add some validation to ensure that "{ecosystem}" is present.